libyul/backends/evm/EVMDialect.cpp

```diff
@@ -385,29 +407,47 @@ EVMDialect::EVMDialect(langutil::EVMVersion _evmVersion, std::optional<uint8_t>
 	m_evmVersion(_evmVersion),
 	m_eofVersion(_eofVersion),
 	m_functions(createBuiltins(_evmVersion, _eofVersion, _objectAccess)),
+	m_verbatimFunctions({}),  // default-initialize all verbatim function pointers to null
```

Comment:
Well, I think `array<T, N>` is POD iff `T` is POD. Since this is array of `unique_ptr`, it's not POD. But it does have aggregate initialization and since unique pointers default-initialize to null, it all works out in the end regardless. I have added the curly braces regardless, they dont hurt and I think they make the matter a bit clearer. Or what do you think?

Reply:
Sure, I don't mind the braces in declaration. It just looked unusual on the initializer list and the comment made it look unnecessary.